python/ray/experimental/channel/nccl_group.py

```diff
@@ -103,6 +146,20 @@ def __init__(
                 cuda_stream, device_id=device.index
             )
 
+            if use_communication_streams:
+                import torch
+
+                self._send_stream = cp.cuda.ExternalStream(
+                    torch.cuda.Stream().cuda_stream, device_id=device.index
+                )
+                self._recv_stream = cp.cuda.ExternalStream(
+                    torch.cuda.Stream().cuda_stream, device_id=device.index
+                )
+                event = cp.cuda.Event()
+                event.record(cp.cuda.get_current_stream())
+                self._send_stream.wait_event(event)
```

Comment:
Why does `send_stream` and `recv_stream` need to wait event on the "current stream" (compute stream?)?

Reply:
check whether this is strictly necessary

Reply:
Removed.